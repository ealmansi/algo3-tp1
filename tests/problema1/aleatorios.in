# D n d1 d2 ... dn
# ... mÃ¡s instancias
# 0

51 1 15
67 2 4 24
10 3 47 11 76
45 4 29 32 53 41
49 5 24 23 10 15 22
7 6 26 57 33 30 10 40
19 7 41 62 93 92 30 12 49
6 8 74 48 17 50 46 99 82 99
40 9 83 75 63 93 90 85 53 17 94
86 10 99 57 78 18 98 93 63 43 23 27
0