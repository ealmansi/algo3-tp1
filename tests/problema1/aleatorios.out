15 1
4 2
11 1
29 4
10 5
26 2
12 2
46 3
63 7
18 10